### DISCUSSION TITLE
When will Daily Challenges for other modes avaliable?

### DISCUSSION THREAD

**Discussion by MizuIro-H** [ppy/osu#31334] [Ideas] [answered]
Now Daily Challenges is only avaliable in standard mode.
If not possible every day, make it one or several days a week is also a good idea, since player number is not as large as std.
(4k/7k should be two modes in mania btw)
<img width="362" alt="スクリーンショット 2024-12-29 11 38 08" src="https://github.com/user-attachments/assets/45b7df60-ffe4-44dd-ae54-f95d512153b1" />

**Comment by peppy** [comment, marked as answer]
We have occasionally run other rulesets in daily challenge. As for when/if it will become available permanently, that's undetermined. It's not zero-effort to run.

**Comment by ItsCollector** [comment]
I am hoping for this to be added too
